articles/api/v1/reference.md

```diff
@@ -222,39 +234,39 @@ The `options` object returned in the `connection` will be different for each str
 		scope: []
 	}
 
-> To obtain `client_id` and `client_secret` for Microsoft Accounts, see [here](/ms-account-clientid).
+To obtain `client_id` and `client_secret` for Microsoft Accounts, see [ms-account-clientid](/ms-account-clientid).
```

Comment:
I'm not sure I like this. I agree using [here] is bad, but it might be better to say something like "see the [Microsoft Account Client ID]() article." or something.